### PR TITLE
[CDF-21862] 😏Import transformation-cli

### PR DESCRIPTION
**Pull request by doctrino**
# Description

image

image

## Checklist

- [ ] Tests added/updated.
- [ ] Run Demo Job Locally.
- [ ] Documentation updated.
- [ ] Changelogs updated in [CHANGELOG.cdf-tk.md](https://github.com/cognitedata/toolkit/blob/main/CHANGELOG.cdf-tk.md).
- [ ] Template changelogs updated in [CHANGELOG.templates.md](https://github.com/cognitedata/toolkit/blob/main/CHANGELOG.templates.md).
- [ ] Version bumped.
  [_version.py](https://github.com/cognitedata/toolkit/blob/main/cognite/cognite_toolkit/_version.py) and
  [pyproject.toml](https://github.com/cognitedata/toolkit/blob/main/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

File: cognite_toolkit/_cdf_tk/commands/featureflag.py [hunk 1]
```diff
@@ -61,7 +62,7 @@ def is_enabled(flag: str | Flags) -> bool:
     @lru_cache
     def to_flag(flag: str) -> Flags | None:
         try:
-            return Flags[flag.upper()]
+            return Flags[flag.upper().replace("-", "_")]
```
Comment (doctrino) [inline cognite_toolkit/_cdf_tk/commands/featureflag.py:65]: It feels more natural for met to write hyphen in a terminal than `_`, so I think I am not the only one this applies to.
```
cdf-tk features set import-cmd --enable`
```

Comment (ronpal) [reply cognite_toolkit/_cdf_tk/commands/featureflag.py:65]: Yes, me too 😆

File: cognite_toolkit/_cdf_tk/prototypes/commands/import_.py [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+        if "action" in transformation:
+            transformation["conflictMode"] = transformation.pop("action")
+        if "ignoreNullFields" not in transformation:
+            # This is required by the API, but the transformation-cli sets it to true by default.
+            transformation["ignoreNullFields"] = True
+        query_path: Path | None = None
+        if isinstance(transformation["query"], dict):
+            query = transformation.pop("query")
+            if "file" in query:
+                query_path = Path(query.pop("file"))
```
Comment (ronpal) [inline cognite_toolkit/_cdf_tk/prototypes/commands/import_.py:128]: Seems you are writing the SQL file path "toolkit-style" above (L79-80), but then you set the query path to whatever was in the manifest here?

**Review by ronpal — APPROVED** [review]
🚋

**Pull request merged**